index.html

```diff
@@ -1,10 +1,167 @@
 <!DOCTYPE html>
 <html lang="en">
 <head>
-  <meta charset="UTF-8">
-  <title>Title</title>
+    <meta charset="UTF-8">
+    <title>Business Web-Opdracht</title>
+    <link rel="stylesheet" href="styles.css">
 </head>
 <body>
+<!--Header-->
+<header class="outer-container-purple">
+    <nav>
+        <ul>
+            <li><a href="#">Case Studies</a></li>
+            <li><a href="#">Services</a></li>
+            <li><a href="#">Contact US</a></li>
+        </ul>
+    </nav>
+
+    <div class="inner-container-short">
+        <h1>Your website is an investment in the success of your business.</h1>
+        <p class="small-weight-p">We are here to help you succeed.</p>
+        <button>Let's work together</button>
+        <img src="assets/squiggle_line_stripe.svg" alt="squiggle line" >
+    </div>
+
+    <div class="seprator"></div>
+
+</header>
+
+<!--main-->
+
+
+<main>
+
+<section class="section-intro"> <!--section intro-->
+    <p id="bg-hello">hello</p>
+    <div class="inner-container-short">
+        <h2>Rekupe is a digital agency in Los Angeles.</h2>
```

Comment:
`h2` is meestal gereserveerd voor section-titels (zoals jouw bg-hello) dus hier kun je beter h3 of lager van maken